src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/SetAzureStorageAccount.cs

```diff
@@ -123,6 +124,34 @@ public bool EnableHttpsTrafficOnly
 
         private bool? enableHttpsTrafficOnly = null;
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipeline = true,
+            HelpMessage = "Storage Account Encryption KeySource.")]
+        public PSKeySource KeySource
```

Comment:
Add the Keycource paramters (4 parameters) to the Set Account cmdelt is one of my original design, but feature team not choose it. this is because:
1. The Set account cmdlets already have so many parameters, and all of them can be set together, if we seperate them by parameter set, it will block user to set it with other setting.
2. We orignally would like to allow set Key Source in both new/Set Account cmdlets (anyway, finally find can't set KEysource in new account.)